src/main/java/cms/gov/madie/measure/utils/MeasureUtil.java

```diff
@@ -26,6 +27,7 @@
 public class MeasureUtil {
   private final CqlDefinitionReturnTypeService cqlDefinitionReturnTypeService;
   private final CqlObservationFunctionService cqlObservationFunctionService;
+  private ValidLibraryNameValidator validLibraryNameValidator = new ValidLibraryNameValidator();
```

Comment:
```suggestion
  private ValidLibraryNameValidator validLibraryNameValidator;
```
Spring takes care of the object initialization since `ValidLibraryNameValidator` was configured as a Bean in the Spring Config and this class is configured as a Service.